static/app/components/profiling/flamegraph.tsx

```diff
@@ -197,7 +205,7 @@ function Flamegraph(props: FlamegraphProps): ReactElement {
       [flamegraphCanvasRef, flamegraphOverlayCanvasRef],
       () => {
         const bounds = flamegraphCanvasRef.getBoundingClientRect();
-        setCanvasBounds(new Rect(bounds.x, bounds.y, bounds.width, bounds.height));
```

Comment:
since setState is not synchronous, this ended up briefly causing the tooltip to render the content with old bounds before these could have been updated